api/handlers.go

```diff
@@ -55,7 +59,79 @@ func (s *Server) Pulse(ctx echo.Context, pulseNumber server.PulseNumberPathParam
 }
 
 func (s *Server) JetDropsByPulseNumber(ctx echo.Context, pulseNumber server.PulseNumberPathParam, params server.JetDropsByPulseNumberParams) error {
-	panic("implement me")
+	if !pulse.IsValidAsPulseNumber(int(pulseNumber)) {
+		response := server.CodeValidationError{
+			Code:               NullableString("400"),
+			Description:        NullableString("Invalid pulse number"),
```

Comment:
Shouldn't we write this message to `ValidationFailures.FailureReason` field? And `pulse_number` in `ValidationFailures.Property`?

Reply:
don't know actually but done